From: David Milburn <dmilburn@redhat.com>
Date: Thu, 18 Nov 2010 20:40:50 -0500
Subject: [char] watchdog: TCO Watchdog for Intel Patsburg Devices
Message-id: <1290112856-1796-4-git-send-email-dmilburn@redhat.com>
Patchwork-id: 29504
O-Subject: [RHEL5.6 PATCH 3/9] BZ 570868 watchdog: TCO Watchdog patch for Intel
	Patsburg DeviceIDs
Bugzilla: 570868
RH-Acked-by: Stanislaw Gruszka <sgruszka@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Michal Schmidt <mschmidt@redhat.com>

commit cad0df370048f7980d4cbc9e88b74db0d87cfc56
Author: Seth Heasley <seth.heasley@intel.com>
Date:   Thu Sep 9 09:58:03 2010 -0700

    watchdog: iTCO_wdt: TCO Watchdog patch for Intel Patsburg DeviceIDs

    This patch adds the Intel Patsburg (PCH) DeviceIDs for iTCO Watchdog.

BZ 570868

diff --git a/drivers/char/watchdog/i8xx_tco.c b/drivers/char/watchdog/i8xx_tco.c
index eb9cba4..6518f15 100644
--- a/drivers/char/watchdog/i8xx_tco.c
+++ b/drivers/char/watchdog/i8xx_tco.c
@@ -146,6 +146,7 @@ enum iTCO_chipsets {
 	TCO_CPT29,	/* Cougar Point */
 	TCO_CPT30,	/* Cougar Point */
 	TCO_CPT31,	/* Cougar Point */
+	TCO_PBG,	/* Patsburg */
 };
 
 static struct {
@@ -233,6 +234,7 @@ static struct {
 	{"Cougar Point", 2},
 	{"Cougar Point", 2},
 	{"Cougar Point", 2},
+	{"Patsburg", 2},
 	{NULL, 0}
 };
 
@@ -348,6 +350,7 @@ static struct pci_device_id iTCO_wdt_pci_tbl[] = {
 	{ ITCO_PCI_DEVICE(0x1c5d,				TCO_CPT29)},
 	{ ITCO_PCI_DEVICE(0x1c5e,				TCO_CPT30)},
 	{ ITCO_PCI_DEVICE(0x1c5f,				TCO_CPT31)},
+	{ ITCO_PCI_DEVICE(0x1d40,				TCO_PBG)},
 	{ 0, },			/* End of list */
 };
 MODULE_DEVICE_TABLE(pci, iTCO_wdt_pci_tbl);
